MusicHub/antivirusProvider/service.py

```diff
@@ -0,0 +1,78 @@
+import json
+from time import sleep
+
+import requests
+
+from MusicHub.antivirusProvider.custom_exception import CustomAntiVirusException
+
+from ..config.settings import Common
+
+
+class AntivirusScan:
+    def __init__(self):
+        self.file_url = "https://api.metadefender.com/v4/file/"
+        self.post_headers = {
+            "Content-Type": "application/octet-stream",
+            "apikey": Common.ANTIVIRUS_API_KEY,
+        }
+        self.get_headers = {
+            "apikey": Common.ANTIVIRUS_API_KEY,
+            "x-file-metadata": "1",
+        }
+        self.bad_status = (
+            1,  # "Infected/Known",
+            2,  # "Suspicious",
+            18,  # "Potentially Vulnerable File",
```

Comment:
enum is better than comments

Reply:
enum AntivirusResponseStatuses = {
   INFECTED = 1,
   SUSPICOUS = 2
}

BAD_STATUSES = [AntivirusResponseStatuses.INFECTED, AntivirusResponseStatuses.SUSPICIOUS]
POTENTIAL_BAD =